workflows/pipe-common/scripts/autoscale_sge.py

```diff
@@ -889,6 +936,7 @@ def _retrieve_preference(pipe, preference, default_value):
     instance_type = os.environ['instance_size']
     instance_image = os.environ['docker_image']
     price_type = os.environ['price_type']
+    instance_cores = int(os.environ['CLOUD_PIPELINE_NODE_CORES'])
```

Comment:
If env not set - use cores count from OS

Reply:
Done in d64e1e5.